blackbox/docs/configuration.txt

```diff
@@ -1852,6 +1852,23 @@ The meaning of the fields of the are as follows:
   | If no protocol is specified, then this entry will be valid for all
   | protocols that rely on host based authentication see :ref:`auth_trust`).
 
+**auth.host_based.config.${order}.require_ssl**
+  | *Runtime:* ``no``
+  | *Default:* ``optional``
+
+  | Specifies whether the client must use SSL/TLS to connect to the cluster.
+  | If set to ``true`` then the client must be connected through SSL/TLS
+  | otherwise is not authenticated. If set to ``false`` then the client must
+  | *not* be connected via SSL/TLS otherwise is not authenticated. Finally
+  | ``optional``, which is the value when the option is completely skipped,
+  | means that the client can be authenticated regardless of SSL/TLS is
+  | used or not.
+
```

Comment:
wouldn' it make more sense to call it just "ssl" because according to this para it sounds like a matcher (like the others like "address" are also), which is fine but a matcher called "require_ssl" sounds odd. could also be ssl='on'